src/main/java/com/project/controller/PasswordResetController.java

```diff
@@ -0,0 +1,65 @@
+package com.project.controller;
+
+import com.project.exception.UserNotFoundException;
+import com.project.model.User;
+import com.project.model.request.PasswordResetRequest;
+import com.project.model.response.GenericResponse;
+import com.project.service.EmailService;
+import com.project.service.PasswordResetService;
+import com.project.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+import java.util.UUID;
+
+@RestController
+@CrossOrigin(origins = "*")
+@RequestMapping("api/users")
+public class PasswordResetController {
+
+    private final UserService userService;
+    private final EmailService emailService;
+    private final PasswordResetService passwordResetService;
+
+    @Autowired
+    public PasswordResetController(UserService userService, EmailService emailService, PasswordResetService passwordResetService){
+        this.userService = userService;
+        this.emailService = emailService;
+        this.passwordResetService = passwordResetService;
+    }
+
+    @GetMapping("/changePassword")
```

Comment:
Please rename your URI to something like `change-password` because this does not comply with Open API guidelines.

Reply:
URLs renamed from camelCase to kebab-case